applications/nrf5340_audio/src/bluetooth/bt_stream/unicast/Kconfig

```diff
@@ -0,0 +1,75 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+rsource "Kconfig.defaults"
+
+menu "Unicast"
+
+choice BT_BAP_UNICAST_BAP_CONFIGURATION
+	prompt "Unicast codec configuration"
+	depends on TRANSPORT_CIS
+	default BT_BAP_UNICAST_CONFIGURABLE
+	help
+	  Select the unicast codec configuration as given in
+	  Table 5.2 of BAP Specification.
+	  USB only supports 48 kHz samplig rate.
+
+config BT_BAP_UNICAST_CONFIGURABLE
+	bool "Configurable unicast settings"
+	depends on TRANSPORT_CIS
+	help
+	  Configurable option that doesn't follow any preset. Allows for more flexibility.
+
+config BT_BAP_UNICAST_16_2_1
+	bool "16_2_1"
+	depends on TRANSPORT_CIS
+	help
+	  Unicast mandatory codec capability 16_2_1
+	  16kHz, 32kbps, 2 retransmits, 10ms transport latency, and 40ms presentation delay.
+
+config BT_BAP_UNICAST_24_2_1
+	bool "24_2_1"
+	depends on TRANSPORT_CIS
+	help
+	  Unicast codec capability 24_2_1
+	  24kHz, 48kbps, 2 retransmits, 10ms transport latency, and 40ms presentation delay.
+endchoice
+
+choice	BT_AUDIO_PRES_DLY_SRCH
+	prompt "Default search mode for the presentation delay"
+	default BT_AUDIO_PRES_DELAY_SRCH_PREF_MIN
+	help
+	  Set the default search mode for the presentation delay.
+
+config BT_AUDIO_PRES_DELAY_SRCH_MIN
+	bool "Largest minimum delay over all audio receivers"
+	help
+	  Search for the largest minimum delay over all audio receivers.
+
+config BT_AUDIO_PRES_DELAY_SRCH_MAX
+	bool "Smallest maximum delay over all audio receivers"
+	help
+	  Search for the smallest maximum delay over all audio receivers.
+
+config BT_AUDIO_PRES_DELAY_SRCH_PREF_MIN
+	bool "Largest minimum preferred delay over all audio receivers"
+	help
+	  Search for the largest minimum preferred delay over all audio receivers.
+
+config BT_AUDIO_PRES_DELAY_SRCH_PREF_MAX
+	bool "Smallest maximum preferred delay over all audio receivers"
+	help
+	  Search for the smallest maximum preferred delay over all audio receivers.
+
+config BT_AUDIO_PRES_DELAY_SRCH_SOURCE
+	bool "Use the audio source/client defined presentation delay"
+	help
+	  Use the audio source/client defined presentation delay if within
+	  AUDIO_MIN_PRES_DLY_US and AUDIO_MAX_PRES_DLY_US range. This will
+	  override the audio receivers presentation delay as long as it is in range of
+	  the max and min supported by the audio receivers. If it is outside this range,
+	  then it will revert to the closest supported value.
+endchoice
+endmenu # Unicast
```

Comment:
Log levels should also be here?

Reply:
Hmm, yes. I don't see it anywhere, so I don't think they were there originally :O

Reply:
Changed it